ci_framework/roles/set_openstack_containers/molecule/default/converge.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
+    cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+    make_openstack_wait_dryrun: true
+    cifmw_install_yamls_defaults:
+      OPERATOR_NAMESPACE: openstack-operator
+  pre_tasks:
+    - name: Run make openstack
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
```

Comment:
err... might need to push the generated role in an actual ansible-compliant location, such as
`{{ ansible_user_dir }}/.ansible/roles` (to be done in the `prepare.yml`):

```
ERROR! the role 'install_yamls_makes' was not found in [...]
```

Known paths are:
```
/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles/set_openstack_containers/molecule/default/roles
/home/zuul/.cache/ansible-compat/387ba3/roles
/home/zuul/.cache/molecule/set_openstack_containers/default/roles
/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles
/home/zuul/.ansible/roles <---------- probably the best location
/usr/share/ansible/roles
/etc/ansible/roles
/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles/set_openstack_containers/molecule/default
/home/zuul/zuul-jobs/roles:
/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles/set_openstack_containers/molecule/default
```